guides/common/modules/proc_creating-a-full-host-boot-disk.adoc

```diff
@@ -0,0 +1,62 @@
+[id="creating-a-full-host-boot-disk"]
+= Creating a full host boot disk
+
+You can generate a full boot disk for a specific host.
+
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-full-host-boot-disk[].
+
+.Prerequisites
+* You have an activation key.
+include::snip_prerequisite-project-client-repository-ak.adoc[]
+
+.Procedure
+include::snip_steps-create-a-host-tab-host.adoc[]
+include::snip_steps-create-a-host-tab-interfaces.adoc[]
+. Click the *Operating System* tab, and verify that all fields contain values.
+Confirm each aspect of the operating system.
+. Click *Resolve* in *Provisioning Templates* to check the new host can identify the right provisioning templates to use.
++
+For more information about associating provisioning templates, see xref:Associating_Templates_with_Operating_Systems_{parent-context}[].
+ifdef::katello,satellite,orcharhino[]
+. Click the *Parameters* tab and ensure that the `kt_activation_keys` parameter exists and provides an activation key.
+If not, add an activation key.
+endif::[]
+. Click *Submit* to save the host details.
+This creates a host entry and the host details page appears.
+. Download the boot disk from {ProjectServer}.
+On the host details page, click the vertical ellipsis and select *Full host '_My_Host_Name_' image*.
+
+[id="cli-creating-a-full-host-boot-disk"]
+.CLI procedure
+. Create a host:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer host create \
+--build true \
+--enabled true \
+--hostgroup "_My_Host_Group_" \
+--location "_My_Location_" \
+--mac "_My_MAC_Address_" \
+--managed true \
+--name "_My_Host_Name_" \
+--organization "_My_Organization_"
+----
+. Ensure that network interface options on your host are set:
++
+[options="nowrap" subs="+quotes"]
+----
+$ hammer host interface update \
+--host "_My_Host_Name_" \
+--managed true \
+--primary true \
+--provision true
+----
+. Download the *Full host image* boot disk from {ProjectServer}:
```

Comment:
Same as above.

Reply:
I don't think the user can download boot disks from SmartProxy directly. @stejskalleos Can you confirm?